packages/auth/src/platform_browser/strategies/popup.ts

```diff
@@ -52,6 +52,7 @@ export const enum _Timeout {
   AUTH_EVENT = 2000
 }
 export const _POLL_WINDOW_CLOSE_TIMEOUT = new Delay(2000, 10000);
+export const _POLL_WINDOW_CLOSE_TIMEOUT2 = new Delay(2000, 10000);
```

Comment:
Is this line needed?

Reply:
Fixed.. that was a change added to trigger CI. Thanks!